WebHostLib/templates/playerOptions/playerOptions.html

```diff
@@ -93,8 +93,8 @@ <h1>Player Options</h1>
                                         {% elif issubclass(option, Options.FreeText) %}
                                             {{ inputs.FreeText(option_name, option) }}
 
-                                        {% elif issubclass(option, Options.ItemDict) and option.verify_item_name %}
-                                            {{ inputs.ItemDict(option_name, option) }}
+                                        {% elif issubclass(option, Options.OptionCounter) and option.valid_keys %}
```

Comment:
This change from verify_item_name to valid_keys makes StartInventory no longer appear anywhere

Reply:
Gonna have to investigate that, because I thought `verify_item_names` makes it **build** a valid_keys from the metaclass

Reply:
Well, it did not work the way I thought it did. StartInventory should be back & code is now more flexible but also significantly uglier :/